components/suggest/src/error.rs

```diff
@@ -29,15 +32,35 @@ pub(crate) enum Error {
 #[derive(Debug, thiserror::Error)]
 #[non_exhaustive]
 pub enum SuggestApiError {
+    #[error("Network error: {reason}")]
+    NetworkError { reason: String },
```

Comment:
```suggestion
    Network { reason: String },
```

Question: Should we adopt the `Error` suffix for our error variants, or drop it?

I noticed some of our crates have it, and other don't, so I went with "no suffix", on the grounds that `SuggestApiError.OtherError` felt like one `Error` too many. But as long as we're consistent, either one sounds good to me!